src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,45 @@
-export const Tabs = () => {};
+import className from "classnames";
+import { Tab } from "../../Tab";
+
+type Props = {
+  tabs: Tab[];
+  active: Tab;
+  selected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({ tabs, active, selected }) => {
+  const handleTab = (tab: Tab) => {
+    if (active.id !== tab.id) {
+      selected(tab);
+    }
```

Comment:
You already have handleSelectTab function in App components, so move this condition there, and delete handleTab function.